scripts/gce_integration_test_metadata.in

```diff
@@ -0,0 +1,24 @@
+#cloud-config
```

Comment:
This file should move to `test-applicaiton/src/main/gce/` following the pattern for kubernetes test.